CMS/settings/base.py

```diff
@@ -128,11 +128,11 @@
     DATABASES = {
         'default': {
             'ENGINE': 'django.db.backends.postgresql',
-            'NAME': os.environ.get('DBNAME'),
-            'HOST': os.environ.get('DBHOST'),
-            'USER': os.environ.get('DBUSER'),
-            'PORT': os.environ.get('DBPORT'),
-            'PASSWORD': os.environ.get('DBPASSWORD')
+            'NAME': os.environ.get('DBNAME', 'phe'),
+            'HOST': os.environ.get('DBHOST', 'host.docker.internal'),
+            'USER': os.environ.get('DBUSER', 'patrick.fleming'),
```

Comment:
There seems something wrong about putting unique personal config in the codebase - it won't work if there are two people working on the project - perhaps this PR isn't the time to fix it though